packages/truffle-core/lib/templates/example.js

```diff
@@ -1,3 +1,5 @@
+var Test = artifacts.require('Test')
```

Comment:
At a glance, I think maybe this should be:
```suggestion
var Example = artifacts.require('Example')
```
Will have to circle back and review the `truffle create` flow

Reply:
I think you are right. What I get for making PR at 2 AM. Will make changes now